src/lib/mina_ledger/test/helpers/ledger_helpers.ml

```diff
@@ -16,11 +16,13 @@ let bin_log n =
   in
   find 0
 
-let ledger_of_accounts accounts =
+let ledger_of_accounts ?depth accounts =
```

Comment:
Shouldn't depth be required? It's silly we were ever inferring it. Or maybe it can have a type like `Fixed_depth of int | Inferred_depth`?

Reply:
That's effectively what this does :sweat_smile: